packages/ag-charts-enterprise/src/features/context-menu/contextMenuItem.ts

```diff
@@ -54,6 +61,28 @@ export function expandBuiltin(
     }
 }
 
+export function expandItems(
+    showing: AgContextMenuItemShowOn,
+    registry: _ModuleSupport.ContextMenuRegistry,
+    items: readonly Readonly<AgContextMenuItem>[],
+    result: ContextMenuItem[]
+) {
+    for (const item of items) {
+        if (typeof item === 'string') {
+            expandBuiltin(showing, registry, item, result);
+        } else {
+            const menuItem = appendItem(showing, item, result);
+            if (item.items && menuItem && menuItem.type === 'submenu') {
+                expandItems(showing, registry, item.items, menuItem.items);
+            }
+        }
+    }
+    // remove trailing 'separator' menu item
+    if (result[result.length - 1]?.type === 'separator') {
+        result.length = result.length - 1;
```

Comment:
```suggestion
        result.pop()
```

A bit more idiomatic